neuland/digitizing/R3BDigitizingPaddle.h

```diff
@@ -39,8 +39,17 @@ namespace R3B::Digitizing
         double energy{};
         double time{};
         double position{};
-        const Channel::Signal& leftChannel;
-        const Channel::Signal& rightChannel;
+        const Channel::Signal leftChannel;
+        const Channel::Signal rightChannel;
+        Channel::CalSignal leftChannelCal;
+        Channel::CalSignal rightChannelCal;
+
+        explicit PaddleSignal(LRPair<const Channel::CalSignal> p_signals)
+            : leftChannelCal{ p_signals.left }
+            , rightChannelCal{ p_signals.right }
+        {
+        }
```

Comment:
Same here. The constructor copies the CalSignals. COPY should always be avoided unless the size of the copied value is less than 8 bytes.